packages/cardano-services/src/InMemoryCache/InMemoryCache.ts

```diff
@@ -50,7 +55,8 @@ export class InMemoryCache {
    *
    * @param key Cache key
    * @param value A value to cache
-   * @param ttl The time to live in seconds.
+   * @param ttl Cache duration in seconds
```

Comment:
"cache duration" was the original terminology that was replaced with TTL. Let's revert this and change the other docblocks to match

Reply:
I lost this info. Thank you for the repetition.

797d581da should solve this.